index.html

```diff
@@ -1781,8 +1786,8 @@ <h3>Processing capabilities</h3>
   to an indexed property in <var>all first match capabilities</var>:
   <ol>
    <li><p>Let <var>validated capabilities</var> be the result
-    of <a>trying</a> to <a>validate capabilities</a> with argument
-    <var>first match capabilities</var>.
+    of <a>trying</a> to <a>validate capabilities</a> with arguments
+    <var>first match capabilities</var> and <var>flags</var>.
```

Comment:
We pass here `flags` to `validate capabilities` algorithm, but it looks like it's not used there

Reply:
On a second thought, maybe we actually want to update the validation algorithm to throw an invalid argument error if let's say classic capabilities are passed, but flags don't contain `http`?

Reply:
I've updated it so we only process those capabilities for the `http` case; that should cause the session to be rejected if they are passed to BiDi.